addons/repos/main.yaml

```diff
@@ -19,6 +19,7 @@ packages:
   - name: gatekeeper
     versions:
       - 3.2.3
+      - 3.7.0
```

Comment:
It's not necessary to update this. It will get changed in another PR.
```suggestion
```

Reply:
Thanks. I was wondering about some of the "doc" type things